### DISCUSSION TITLE
Remind inside Infoboxes

### DISCUSSION THREAD

**Discussion by bubugian** [XCSoar/XCSoar#1372] [General]
Hi all,
is there a way to set one or more infoboxes with a predefined (and fixed) "text reminder"? 
Something like "arrival reference is mr. Mike" or a typical glider speed or something else..

The only way I found to have a text note is to write the xcsoar-checklist.txt file. 
Are there other ways ?

Thanks bye !

**Comment by lordfolken** [comment]
The checklist would be the place.

You can create multiple pages, by prefixing the title with brakets

[Page 1]
Item 1
Item 2

[Page 2]
Item 1..


My personal checklist starts with
* things to check at home
* packlist
* rigging
* prelaunch
* takeoff
* ATC interactions
* mindset
* landing
* post lalanding

**Reply by bubugian** [reply]
Thanks, I appreciate the multiple page suggestion !
